x/concentrated-liquidity/README.md

```diff
@@ -1155,6 +1155,22 @@ x/twap module).
 Lastly, see the "Listeners" section for more details on how twap is enabled by the use of
 these hooks.
 
+## Parameters
+
+- `AuthorizedQuoteDenoms` []string 
+
+This is a list of quote denoms that can be used as token1 when creating a pool.
+We limit the quote assets to a small set for the purposes of having convinient price increments stemming
+from tick to price conversion. These increments are in a human readable magnitude only for token1 as a quote.
+For limit orders in the future, this will be a desirable property in therms of UX as to allow users
+to set limit orders at prices in terms of token1 (quote asset) that are easy to reason about.
+
+This goes in-hand with centralized exchanges such as Coinbase that limit the quote asset set
+to only a few denoms.
+
+Our list at launch is expected to consist of OSMO, DAI and USDC. These are set in the v16
+upgrade handler.
```

Comment:
Potentially update this if you decide to go with WBTC and WETH

Reply:
Suggested for this to be discussed separately since these have already been ACKed by Sunny